tfjs-converter/src/executor/graph_executor.ts

```diff
@@ -247,6 +276,7 @@ export class GraphExecutor implements FunctionExecutor {
                 `Please use model.executeAsync() instead.`);
           }
           tensorsMap[node.name] = tensors;
+          this.keepTensors(tensors, this.tensorsPendingDisposal);
```

Comment:
```
if (this.keepIntermediateTensors)
{
  this.keepTensors(tensors, this.tensorsPendingDisposal);
}

Reply:
keepTensors is being call at two place, so I move this conditional inside keepTensors,